server/rating_service/rating_service.py

```diff
@@ -0,0 +1,382 @@
+from typing import Dict
+from .typedefs import (
+    TeamRatingData,
+    GameRatingData,
+    GameRatingSummary,
+    PlayerID,
+    ServiceNotReadyError,
+    EntryNotFoundError,
+)
+
+import asyncio
+
+from server.db import FAFDatabase
+from server.core import Service
+from server.player_service import PlayerService
+from server.decorators import with_logger
+from server.metrics import rating_service_backlog
+import server.config as config
+
+from server.games.game_results import GameOutcome
+
+from server.rating import RatingType
+from trueskill import Rating
+from .game_rater import GameRater, GameRatingError
+
+from sqlalchemy import select, and_
+from server.db.models import legacy_ladder1v1_rating as legacy_ladder1v1_table
+from server.db.models import legacy_global_rating as legacy_global_table
+from server.db.models import leaderboard_rating as rating_table
+from server.db.models import leaderboard as rating_type_table
+from server.db.models import game_player_stats as gps_table
```

Comment:
I feel like these renames make the queries more confusing since you have to go back and check what table they're actually accessing.